src/app/settings/components/settings/settings-menu/settings-menu.component.ts

```diff
@@ -1,11 +1,33 @@
-import { Component } from '@angular/core'
+import { Component, OnDestroy, OnInit } from '@angular/core'
+import { NavigationEnd, Router } from '@angular/router'
+import { Subject, takeUntil } from 'rxjs'
 import { SettingsPaths } from 'src/app/settings/settings-routing.module'
+import { ShelfPaths } from 'src/app/settings/shelf/shelf.module'
 
 @Component({
   selector: 'sofie-settings-menu',
   templateUrl: './settings-menu.component.html',
   styleUrls: ['./settings-menu.component.scss'],
 })
-export class SettingsMenuComponent {
+export class SettingsMenuComponent implements OnInit, OnDestroy {
   public settingsPaths = SettingsPaths
+  public shelfPaths = ShelfPaths
```

Comment:
These are enums you are exposing to the HTML. Be nice if those were PascalCased i.e. `public SettingsPaths = SettingsPaths`.